### ISSUE TITLE
Uncaught ReferenceError: f is not defined

### ISSUE THREAD

**Issue by nicoxxxcox** [Octagon-simon/octaValidate#6]
Console return error `Uncaught ReferenceError: f is not defined` when : 
- I set a minlength on an input
- I forgot an to set an id on all inputs

It stop at <code>throw new Error(`One or more fields in ${f} is missing an Identifier`)</code>

*ps :My instance is in strictMode*

Thanks a lot for this nice lib 🙏  !!!

**Comment by Octagon-simon** [comment]
Wow! Thank you for pointing this bug out, I will resolve it and update the repo as soon as I am done.
I'm glad you loved this library!!! 
Let's keep coding 🚀 🚀 🚀

**Comment by Octagon-simon** [comment]
This bug has been resolved on patch 1.1.2

Update your local copy of the library:

```html
<script src="https://unpkg.com/octavalidate@1.1.2/src/validate.js"></script>
```
